src/plugins/guided_onboarding/public/components/guide_panel.styles.ts

```diff
@@ -43,5 +43,15 @@ export const getGuidePanelStyles = (euiTheme: EuiThemeComputed) => ({
       background: ${euiTheme.colors.ghost};
       padding: 24px 30px;
     `,
+    flyoutFooterLink: css`
+      color: ${euiTheme.colors.darkShade};
```

Comment:
Can the `color` prop be used within the `EuiButtonEmpty` component instead? https://elastic.github.io/eui/#/navigation/button#empty-button

Reply:
Ideally, it would be passed into the prop but the only supported colors are: `"text", "accent", "primary", "success", "warning", "danger", "ghost"`, so because we wanted it to be a gray color I added it to the custom style.